api/src/main/java/org/openmrs/module/mirebalais/MirebalaisHospitalActivator.java

```diff
@@ -147,11 +147,14 @@ public void started() {
                 }
             }
 
-
             if (config.isComponentEnabled(Components.RADIOLOGY) && config.getSite().equals(ConfigDescriptor.Site.MIREBALAIS)) {
                 updateGlobalProperty(OpenmrsConstants.GP_ORDER_NUMBER_GENERATOR_BEAN_ID, MirebalaisConstants.RADIOLOGY_ORDER_NUMBER_GENERATOR_BEAN_ID);
             }
 
+            if (config.getCountry().equals(ConfigDescriptor.Country.MEXICO)) {
+                updateGlobalProperty(OpenmrsConstants.GLOBAL_PROPERTY_DEFAULT_THEME, "orange");
+            }
+
```

Comment:
Interesting... I didn't realize this GP existed, and am interested in seeing the results... PIH abandoned the purple color scheme for orange a while back and we haven't updated the other sites accordingly... if this orange is close enough to the PIH scheme we may want to do it everywhere...

That being said, I would thing that this would live here:

https://github.com/PIH/openmrs-module-pihcore/blob/master/api/src/main/java/org/openmrs/module/pihcore/deploy/bundle/mexico/MexicoMetadataBundle.java